src/main/java/org/reaktivity/nukleus/http2/internal/stream/Http2ServerFactory.java

```diff
@@ -2614,8 +2677,17 @@ private void cleanupBudgetCreditorIfNecessary()
             }
         }
 
+        public void cleanupDataDecode()
+        {
+            dataStreamId = 0;
+            endStream = false;
+            dataLength = 0;
+            dataBytesConsumed = 0;
+        }
```

Comment:
Wouldn't expect this to be necessary if we use `server.decodableDataBytes`, since it will be overwritten on each `H2_DATA` frame decode and unused in between.